packages/lib-utils/src/ws/ws-utils.ts

```diff
@@ -0,0 +1,30 @@
+import { getUtilsConfig } from '../config';
+
+export const createURL = (host: string, path: string): string => {
```

Comment:
Not sure if we're tracking it, but this reminds me that we should have an ESLint rule that consistently enforces an explicit return type for all functions and class methods, as we discussed earlier.

Reply:
I'll make sure one is on the backlog... good recall.

Reply:
Added here: https://issues.redhat.com/browse/HAC-782